pyproject.toml

```diff
@@ -1,6 +1,64 @@
 [build-system]
 requires = [
-    "setuptools>=42",
-    "wheel",
+    "setuptools>=64",
+    "setuptools_scm>=8"
 ]
 build-backend = "setuptools.build_meta"
+
+[project]
+name = "neurom"
```

Comment:
Doesn't that have to be NeuroM. It's project name, not package name. Right?

Reply:
not sure; I've set it to `NeuroM` now.

Reply:
Note that the original has:
``
name='neurom',
```

Reply:
I guess we'll go back to the original